test/external-e2e/run.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Ensure that we have the desired version of the ginkgo test runner.
+
+PROJECT_ROOT=$(git rev-parse --show-toplevel)
+
+install_ginkgo () {
+    apt update -y
+    apt install -y golang-ginkgo-dev
+}
+
+setup_e2e_binaries() {
+    mkdir /tmp/csi-azuredisk
+
+    # download k8s external e2e binary for kubernetes v1.19
+    curl -sL https://storage.googleapis.com/kubernetes-release/release/v1.19.0/kubernetes-test-linux-amd64.tar.gz --output e2e-tests.tar.gz
+    tar -xvf e2e-tests.tar.gz && rm e2e-tests.tar.gz
+
+    # install the azuredisk-csi-driver driver
+    $PROJECT_ROOT/deploy/install-driver.sh
```

Comment:
this will install the master branch driver not current driver with code change, should depend on `make e2e-bootstrap` to install driver.

Reply:
updated.